SecretSanta/src/SecretSanta.Data/User.cs

```diff
@@ -4,21 +4,15 @@
 
 namespace SecretSanta.Data
 {
-    public class User
+    public class User : FingerPrintEntityBase
     {
-        public int Id { get; }
         public string FirstName { get => _FirstName; set => _FirstName = value ?? throw new ArgumentNullException(nameof(FirstName)); }
         private string _FirstName = string.Empty;
         public string LastName { get => _LastName; set => _LastName = value ?? throw new ArgumentNullException(nameof(LastName)); }
         private string _LastName = string.Empty;
-        public ICollection<Gift> Gifts { get; }
-
-        public User(int id, string firstName, string lastName, List<Gift> gifts)
-        {
-            Id = id;
-            FirstName = firstName;
-            LastName = lastName;
-            Gifts = gifts;
-        }
+        public ICollection<Gift> Gifts { get=>_Gifts; }
+        private ICollection<Gift> _Gifts = new List<Gift>();
+        public List<UserGroup> UserGroups { get=>_UserGroups; }
+        private List<UserGroup> _UserGroups = new List<UserGroup>();
     }
```

Comment:
Dont forget to add a santa!
```suggestion
         public User? Santa { get; set; }
    }
```

Reply:
thank you! I somehow missed that!